src/de/grobox/liberario/ui/LocationGpsView.java

```diff
@@ -160,6 +160,7 @@ public void activateGPS() {
 			locationManager.removeUpdates(this);
 			Toast.makeText(getContext(), getContext().getString(R.string.error_no_location_provider), Toast.LENGTH_LONG).show();
 
+			searching = false;
```

Comment:
Otherwise the GpsLocationView "hangs" on searching. It won't search again because searching is true and it stopped searching at the same time. You can reproduce this by doing a normal search with GPS disabled.
I assumed this was unintended behavior. Moreover this allows the search, once started by the appwidget to abort automatically and not hang in the ProcessDialog.

I'm going to add a comment if this stays in the code.